Dockerfile.mysql

```diff
@@ -0,0 +1,8 @@
+ARG DOCKER_TAG=9.1.0
+FROM mysql:${DOCKER_TAG}
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Hardcoding the MySQL root password is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.